build/docker-build-push.yml

```diff
@@ -0,0 +1,30 @@
+# DESCRIPTION: 
+# Builds and pushes a docker image for a given FHIR version
+
+parameters:
+- name: version
+  type: string
+- name: tag
+  type: string
+
+steps:
+- task: DockerCompose@0
+  displayName: 'Build FHIR ${{parameters.version}} Server Image'
+  inputs:
+    action: Build services
+    azureSubscriptionEndpoint: 'docker-build'
+    azureContainerRegistry: 'healthplatformregistry.azurecr.io'
```

Comment:
prefer the variables if it doesn't add any extra complexity :P 
Or, is `azureContainerRegistry` required for the build if the next step is push?

Reply:
Refactored the variables into a template